rfc/0054-sdk3-management-apis.md

```diff
@@ -1090,6 +1090,14 @@ public interface IAnalyticsIndexManager{
      void DisconnectLink(DisconnectLinkAnalyticsOptions options);
 
      map[string]map[string]int GetPendingMutations(GetPendingMutationsAnalyticsOptions options);
+
+     void CreateLink(AnalyticsLink link, CreateLinkAnalyticsOptions options);
+
+     void ReplaceLink(AnalyticsLink link, ReplaceLinkAnalyticsOptions options);
+
+     void DropLink(string linkName, string dataverseName, DropLinkAnalyticsOptions options);
```

Comment:
@chvck I think a common requirement will be for code like this work:

analytics.getAllLinks().foreach(link => analytics.dropLink(link))

E.g. at the start or end of testing.
However the API as-is doesn't support this.  dropLink needs a linkName and dataverseName, and the generic AnalyticsLink interface doesn't provide that.  But, they could - at least with the 3 analytics links we know about currently.